src/components/task-meter/task-meter.ts

```diff
@@ -0,0 +1,130 @@
+import { customElement } from "../../helpers/customElement";
+import { AbstractComponent } from "../../mixins/abstractComponent";
+import { html, HTMLTemplateResult, LitElement, unsafeCSS } from "lit";
+import { repeat } from "lit/directives/repeat.js";
+import { Tag } from "../../models/tag";
+import { consume } from "@lit/context";
+import { gridTileContext, injectionContext } from "../../helpers/constants/contextTokens";
+import {
+  RequiredDecision,
+  requiredNewTagPlaceholder,
+  requiredVerificationPlaceholder,
+  VerificationGridTileContext,
+} from "../verification-grid-tile/verification-grid-tile";
+import { decisionNotRequired } from "../../models/decisions/decisionNotRequired";
+import { VerificationGridInjector } from "../verification-grid/verification-grid";
+import { DecisionOptions } from "../../models/decisions/decision";
+import { ifDefined } from "lit/directives/if-defined.js";
+import taskMeterStyles from "./css/style.css?inline";
+
+/**
+ * @description
+ * A component to display what decisions have been applied to a subject and if
+ * there are any outstanding decisions.
+ */
+@customElement("oe-task-meter")
+export class TaskMeterComponent extends AbstractComponent(LitElement) {
+  public static styles = unsafeCSS(taskMeterStyles);
+  public static tagName = "oe-task-meter";
```

Comment:
pull this out to class constant that is exported. Then reuse it in all three instances in this file.

Ditto for the other place where you've used this pattern